CHANGELOG.md

```diff
@@ -141,6 +141,9 @@
   emitting invalid code
   ([PR #730](https://github.com/jasmin-lang/jasmin/pull/730)).
 
+- Expansion of `#copy` operators uses an intermediate register when needed
+  ([PR #735](https://github.com/jasmin-lang/jasmin/pull/735)).
```

Comment:
There is no issus solved by this PR?

Reply:
No. You can open one if you think it’s necessary.